source/query-with-natural-language/query-with-natural-language.txt

```diff
@@ -0,0 +1,72 @@
+.. _query-natural-language:
+
+===========================
+Query with Natural Language
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You can use MongoDB Compass to ask natural language questions about your
+data. Compass uses AI to generate filter queries and aggregations based 
+on the prompts you provide.
+
+Use Cases
+---------
+
+You may want to use natural language to query in Compass to:
+
+- Ask plain text questions about your data.
+- Create an initial query or aggregation pipeline that you can 
+  modify to suit your requirements.
+- Learn how to write complex queries with multiple aggregation stages.
+
+Behavior
+--------
+
+- Compass natural language querying is available starting in version 
+  ``1.40.x``.
+
+- Natural language querying utilizes `Azure Open AI 
+  <https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+  as its current provider. This provider may be subject to change in 
+  the future.
+
+- The MongoDB Compass natural language querying feature is on a rolling 
+  release schedule, with additional functionality and improvements planned 
+  for release by the end of October. As a result, some users may 
+  temporarily have functionality that other users do not. 
+
+Get Started
+-----------
+
+- :ref:`enable-natural-language-querying`
+- :ref:`prompt-natural-language-query`
+- :ref:`prompt-natural-language-agg`
+
+Details
+-------
+
+When you query your data using natural language in Compass, the text of 
+your prompts and details about your MongoDB schemas are sent to 
+Microsoft and OpenAI for processing. Your data is not stored on any 
+third party storage systems or used to train AI models. 
+
+This software uses generative artificial intelligence. It is 
+experimental and may give inaccurate results. Your use of this software 
+is subject to MongoDB's:
+
+- `Terms of Use <https://www.mongodb.com/legal/terms-of-use>`_
+- `Acceptable Use Policy <https://www.mongodb.com/legal/acceptable-use-policy>`_
+- `Privacy Policy <https://www.mongodb.com/legal/privacy-policy>`_
```

Comment:
Should this block be also wrapped in a UI Note component like on this page? https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-33192/query-with-natural-language/prompt-natural-language-query/#about-this-task

Reply:
This was a deliberate stylistic choice, for the landing I elected for a slightly cleaner look with the same info and without the note directive.